app/utils/trayWithText.js

```diff
@@ -0,0 +1,262 @@
+const mergeImg = require('merge-img-vwv')
+const pathToImages = 'app/images/app-icons/numbers/generated-numbers/'
+const pathToCircleImages = 'app/images/app-icons/round-clock/'
+const log = require('electron-log')
+const baseImages = 'app/images/app-icons/numbers/'
+
+class TrayWithText {
+  iconWithNumber (
+    imagePath,
+    minutesToLongBreak,
+    darkModeString,
+    monochrome,
+    platform,
+    invertedMonochromeString
+  ) {
+    let minutesOnTray = minutesToLongBreak
+    let minutesAsNumber = 99
+    try {
+      minutesAsNumber = Number(minutesOnTray)
+    } catch (er) {
+      minutesAsNumber = 99
+    }
+    if (minutesAsNumber >= 99) {
+      minutesOnTray = '99'
+    }
```

Comment:
Now it could be moved to appIcon. Your old suggestion would make one big folder with imageIcons that would make it hard to know what is where. Currently all images are separated by functionality and it's still sometimes bit to much files in folder. I could copy them  all to one folder but it would not make it clearer. You would get all in one folder, but no. of images will be hard to find anything. So I strongly recommend let current folder structure. In current implementation there is no generation at runtime, so as long as there are files in correct places all works.

Reply:
I don't mind many pics under one folder as I rarely touch them and I expect similar for the future. There is much higher chance that I will be changing `AppIcon` and stuff around icons so better for me to have one class to manage everything

Reply:
I moved all to one class. Still I not moved pictures. Currently in app/images/app-icons is 27 pictures. If I moved all new one there will be 27 + 1000 + 92. Are you 100% sure you want them all in this one folder? This will not be big rework to move them, it will just make finding basic images much harder to find.

Reply:
I moved it all in but this could be of course reverted.
e0830cb

Reply:
I guess all is as expected, if not, let me know what change.